src/layouts/PostFeedLayout/index.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import Link from '../../utils/link';
+import { getComponent } from '../../components-registry';
+import { getBaseLayoutComponent } from '../../utils/base-layout';
+
+export default function PostFeedLayout(props) {
+    const { page, site } = props;
+    const BaseLayout = getBaseLayoutComponent(page.baseLayout, site.baseLayout);
+    const { title, topSections = [], bottomSections = [], pageIndex, baseUrlPath, numOfPages, numOfTotalItems, ...rest } = page;
+    const PostFeedSection = getComponent('PostFeedSection');
+
+    return (
+        <BaseLayout page={page} site={site}>
+            <main id="main" className="layout page-layout">
+                {title && (
+                    <h1 className="sr-only" data-sb-field-path="title">
+                        {title}
+                    </h1>
+                )}
+                {renderSections(topSections, 'topSections')}
+                <PostFeedSection {...rest} />
+                <PageLinks pageIndex={pageIndex} baseUrlPath={baseUrlPath} numOfPages={numOfPages} numOfTotalItems={numOfTotalItems}/>
+                {renderSections(bottomSections, 'bottomSections')}
+            </main>
+        </BaseLayout>
+    );
+}
+
+function renderSections(sections: any[], fieldName: string) {
+    if (sections.length === 0) {
+        return null;
+    }
+    return (
+        <div data-sb-field-path="sections">
+            {sections.map((section, index) => {
+                const Component = getComponent(section.type);
+                if (!Component) {
+                    throw new Error(`no component matching the page section's type: ${section.type}`);
+                }
+                return (
+                    <div key={index} data-sb-field-path={`${fieldName}.${index}`}>
+                        <Component {...section} />
+                    </div>
+                );
+            })}
+        </div>
+    );
+}
+
+function PageLinks({ pageIndex, baseUrlPath, numOfPages, numOfTotalItems }) {
```

Comment:
@smnh I tested the pagination in Studio and it worked. Nice! 

* I tested changing the "Number of Posts per page" to 3 (of 4) and pagination just worked.
* I tested adding new post pages and they worked.